src/test/java/codeu/controller/ProfileServletTest.java

```diff
@@ -27,13 +35,29 @@ public void setup() {
     mockRequestDispatcher = Mockito.mock(RequestDispatcher.class);
     Mockito.when(mockRequest.getRequestDispatcher("/WEB-INF/view/profiles.jsp"))
             .thenReturn(mockRequestDispatcher);
+    mockResponse = Mockito.mock(HttpServletResponse.class);
   }
 
   @Test
   public void testDoGet() throws IOException, ServletException {
     profilePageServlet.doGet(mockRequest, mockResponse);
     Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
-  }
-}
 
+    User testUser = new User(UUID.randomUUID(), "Test User", "Test Password", Instant.now());
```

Comment:
I'm not sure what this test is trying to test? The first line in this function calls the `doGet()` function, and then the second line in this test verifies that the servlet forwards the request to the JSP. But then I think most of these other lines aren't testing anything specific.

Actually are you sure this test passes? I would think that the `setUserStore()` function would need to be called before the `doGet()` function. What happens when you run `mvn test`?

Reply:
It passed the tests, I was trying to create a random User and then see if the Profile matched up